s3config-test.yaml

```diff
@@ -70,6 +70,7 @@ S3_SERVER_CONFIG:                                     # Section for S3 Server
    S3_SERVER_MOTR_ETIMEDOUT_MAX_THRESHOLD: 100          # Number of ETIMEDOUT errors per monitoring window before s3server restart
    S3_SERVER_MOTR_ETIMEDOUT_WINDOW_SEC: 2               # Monitoring window for motr ETIMEDOUT errors in seconds
    S3_SERVER_ENABLE_ADDB_DUMP: true                     # If set to true, then addb dump will be collected
+   S3_MAX_EXTENDED_OBJECTS_IN_FAULT_MODE: 0             # Maximum number of allowed recovery by S3 when fault happens and Motr error out with specific error code. When 0, feature is disabled.
```

Comment:
default will be 1

Reply:
default will be 0 (i.e. feature is disabled)